### DISCUSSION TITLE
Can't upload proof of your academic status

### DISCUSSION THREAD

**Discussion by zAlweNy26** [community/community#36872] [GitHub Education]
I am trying to renew my subscription to the Student Developer Pack, but when I try to upload screenshots it keeps giving me these problems, despite the fact that
- I am close to the university campus (physically)
- I have already completed my GitHub profile years ago
- I have given permission to the site to get the location and webcam (and they do indeed work)
- for some reason it wants me to upload a "Dated School ID" which I don't really know what it is (without the possibility of using the other verification options)

![Request a discount - GitHub Education 2022-10-21 09 07 48](https://user-images.githubusercontent.com/41266342/197134683-8653deef-9de7-4e31-b37d-e169dc5681e8.jpg)

**Comment by kude5n1k** [comment]
Have you seen the info in this post [Apply to GitHub Global Campus as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)

**Reply by zAlweNy26** [reply]
Yes, that's what I did, but instead of taking a photo with my webcam, i uploaded a screenshot of the website page of my university where it says the required informations.
